src/toil_scripts/lib/files.py

```diff
@@ -35,6 +35,13 @@ def move_files(file_paths, output_dir):
         shutil.move(file_path, dest)
 
 
+def move_file_job(job, name, file_id, output_dir):
+    """Job version of `move_files` for a single file"""
```

Comment:
Moar docs plz.

Also, what exactly is the use case for this function? I'll dig back into the diff to see where it is invoked, but as a bystander I'm not entirely understanding the "why" for this function.